### ISSUE TITLE
Console error Paper 1.11

### ISSUE THREAD

**Issue by andris155** [lishid/OpenInv#62]
```
[09:37:44] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to OpenInv v3.1.0
java.lang.NoSuchMethodError: net.minecraft.server.v1_11_R1.World.a(Lnet/minecraft/server/v1_11_R1/AxisAlignedBB;)Z
	at com.lishid.openinv.internal.v1_11_R1.AnySilentContainer.isBlockedShulkerBox(AnySilentContainer.java:125) ~[?:?]
	at com.lishid.openinv.internal.v1_11_R1.AnySilentContainer.isAnyContainerNeeded(AnySilentContainer.java:73) ~[?:?]
	at com.lishid.openinv.listeners.PlayerListener.onPlayerInteract(PlayerListener.java:66) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor188.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:44) ~[patched_1.11.jar:git-Paper-976]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched_1.11.jar:git-Paper-976]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.11.jar:git-Paper-976]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [patched_1.11.jar:git-Paper-976]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) [patched_1.11.jar:git-Paper-976]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:233) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PlayerInteractManager.a(PlayerInteractManager.java:444) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:899) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PlayerConnectionUtils$1.run(SourceFile:13) [patched_1.11.jar:git-Paper-976]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_121]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_121]
	at net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:817) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:403) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:749) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:648) [patched_1.11.jar:git-Paper-976]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
[09:37:45] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to OpenInv v3.1.0
java.lang.NoSuchMethodError: net.minecraft.server.v1_11_R1.World.a(Lnet/minecraft/server/v1_11_R1/AxisAlignedBB;)Z
	at com.lishid.openinv.internal.v1_11_R1.AnySilentContainer.isBlockedShulkerBox(AnySilentContainer.java:125) ~[?:?]
	at com.lishid.openinv.internal.v1_11_R1.AnySilentContainer.isAnyContainerNeeded(AnySilentContainer.java:73) ~[?:?]
	at com.lishid.openinv.listeners.PlayerListener.onPlayerInteract(PlayerListener.java:66) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor188.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:44) ~[patched_1.11.jar:git-Paper-976]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched_1.11.jar:git-Paper-976]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.11.jar:git-Paper-976]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [patched_1.11.jar:git-Paper-976]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) [patched_1.11.jar:git-Paper-976]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:233) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PlayerInteractManager.a(PlayerInteractManager.java:444) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:899) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.PlayerConnectionUtils$1.run(SourceFile:13) [patched_1.11.jar:git-Paper-976]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_121]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_121]
	at net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:817) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:403) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:749) [patched_1.11.jar:git-Paper-976]
	at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:648) [patched_1.11.jar:git-Paper-976]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
```

**Comment by Jikoo** [comment]
Unfortunately, for now you'll either have to downgrade OpenInv to 3.0.5 or update your server to 1.11.2 - the NMS revision remained the same despite multiple NMS changes, and [my attempt to support both](https://github.com/lishid/OpenInv/blob/e34e51d12718e0eef705b169846c0fdfab35ee79/internal/v1_11_R1/src/main/java/com/lishid/openinv/internal/v1_11_R1/AnySilentContainer.java#L123-L129) apparently doesn't work. It's certainly possible, but I don't have the time to push a new release for a while.

**Comment by jjm223** [comment]
@Jikoo I believe you need to catch a Throwable, not just an Exception. If you don't have time, I can submit a PR for this later today.

**Comment by Jikoo** [comment]
>I believe you need to catch a Throwable, not just an Exception

Aah, thank you, for some reason I was thinking it was a NoSuchMethodException and not NoSuchMethodError. If you do want to take care of it, be my guest, but otherwise I should be able to get to in in 8 hours or so.

**Comment by Jikoo** [comment]
I've released [3.1.1](https://github.com/lishid/OpenInv/releases/tag/3.1.1), please let me know if you encounter any further issues. Don't have time for a Bukkit release right now, expect that later today.
